fetch/fetch.go

```diff
@@ -9,7 +9,8 @@ import (
 	"os"
 	"strconv"
 	"strings"
-	"vip/store"
+
+	"github.com/vokal/vip/store"
```

Comment:
Why are these not relative imports again? I fear my local dev environment needs changed big time.